app/frontend/terrier/tabs.ts

```diff
@@ -51,6 +91,26 @@ export class TabContainerPart extends TerrierPart<TabContainerState> {
             this.state.side = m.data.side
             this.dirty()
         })
+
+        this.makePlugin(SortablePlugin, {
+            zoneClass: 'tt-tab-list',
+            targetClass: 'tab',
+            onSorted: (_, evt) => {
+                this.renumberTabs(evt.toChildren)
+            }
+        })
```

Comment:
We should add a new parameter to `TabContainerPart` to determine whether the tabs are reorderable or not, defaulting to false. Tab reordering should be opt-in.